elements/hex-picker/src/hex-picker.js

```diff
@@ -282,14 +312,15 @@ class HexPicker extends LitElement {
     this._dispatchChange(computedHex);
   }
 
-  _dispatchChange(value) {
-    let customEvent = new CustomEvent("value-changed", {
-      bubbles: true,
-      cancelable: false,
-      composed: false,
-      detail: value,
-    });
-    this.dispatchEvent(customEvent);
+  _dispatchChange() {
+    this.dispatchEvent(
+      new CustomEvent("value-changed", {
+        bubbles: true,
+        cancelable: false,
+        composed: false,
+        detail: { value: this },
```

Comment:
Sry here, not the compiled file to fix this

Reply:
Fixed, I guess the PR was automatically updated when I pushed to my fork